src/vmm/src/vstate/vcpu/x86_64.rs

```diff
@@ -254,6 +241,14 @@ impl KvmVcpu {
             .set_cpuid2(&kvm_cpuid)
             .map_err(KvmVcpuConfigureError::SetCpuid)?;
 
+        let mut msrs = vcpu_config.cpu_config.msrs.clone();
+        self.msrs_to_save.extend(msrs.keys());
+
+        // Apply MSR normalization to boot Linux.
```

Comment:
> I am a bit cautious that the normalization term can be overloaded. Can we leave a note somewhere on what exactly we mean by the MSR normalization?

I totally agree with that. It might be better to rephrase it instead of using "normalization" Will do.

Reply:
done